### ISSUE TITLE
Windows on ARM (ARM64/AArch64) Support

### ISSUE THREAD

**Issue by hifihedgehog** [bambulab/BambuStudio#4220]
**Is your feature request related to a problem? Please describe.**
I have an ARM-based Windows machine and would like to use it with Bambu Studio for PC-directed printing. I can run the x86-64 build, but at a tremendous penalty to both performance and battery life. A native ARM build would resolve these issues.

**Describe the solution you'd like**
I'd like an ARM native build of Bambu Studio for Windows. This would put ARM-based hardware for Windows on par with Apple ARM hardware.

**Describe alternatives you've considered**
Running an x86-64 build on ARM is possible, but at a major performance and battery life penalty.

**Additional context**
Qualcomm Snapdragon X series processors are taking the market by storm this month with historic crossroads and massive push from all major PC makers. Microsoft is giving priority development and support to ARM and relegating x86 to second-class citizen.

**Comment by fijsh** [comment]
Yes, this would definitely be great to have!

**Comment by myshophosting** [comment]
I also now run an ARM laptop with Windows 11.. An ARM native version would be awsome.

**Comment by goat77** [comment]
I would also like to see this.

**Comment by BillsWorkbench** [comment]
Yes, please; the intel version is "quick exiting" on my Arm64 Notebook. (snapdragon elite)

**Comment by andrew-c0** [comment]
One up from me as well. Would be great to have native support.

**Comment by danielcherubini** [comment]
Yes please

**Comment by mhertel** [comment]
I would like to see it as well!

**Comment by adachi86** [comment]
+1 (Surface Laptop 7 w/ Snapdragon X Elite)

**Comment by pleite** [comment]
+1 really slow using preview on Surface Pro X (first edition).

**Comment by dirtdiver99** [comment]
+1, this would be amazing.

**Comment by spbrogan** [comment]
Agree.  Please provide native application for this environment.

**Comment by RyanEwen** [comment]
Brand new Surface Laptop 7 here and Bambu Studio closes when launched (seems related o network plugin).

Oddly, Orca Slicer works fine

**Comment by RyanEwen** [comment]
Hmm apparently 1.10.1 works... it's just 1.10.2 that crashes for me.

**Comment by adachi86** [comment]
> Hmm apparently 1.10.1 works... it's just 1.10.2 that crashes for me.

Same

**Comment by ludovandenbrink** [comment]
+1 would love to see this.

**Comment by egeerardyn** [comment]
+1 here

**Comment by rossturk** [comment]
+1

**Comment by AshishMBakshi** [comment]
+1 - currently using 1.10.1.50 on WoA, but a native version of the latest build would be ideal.

**Comment by konashevich** [comment]
AARCH64 need with compatible network plugin! Dear developers, stop ignoring the community! Either issue opens source plugin (accountless for example if you don't to disclose this part of the code, or something) or add support to arm64/aarch64. People ask you for many years! How long will you ignoring it?

**Comment by gauravmahto** [comment]
+1

**Comment by ethanspitz** [comment]
I would love to have this. Just got the H2S and a bit sad about using x64 version of the slicer on my ARM64 laptop.

**Comment by SCIEN-00** [comment]
+1

**Comment by ChristianNewton** [comment]
+1

**Comment by wtahler** [comment]
+1

**Comment by maxnabokow** [comment]
+1

**Comment by lexcyn** [comment]
Has there been any official word on this? We definitely need a native version as performance on the emulated version is really bad

**Comment by mrgrzanka** [comment]
+!

**Comment by marcelecke97** [comment]
+1

**Comment by blauver** [comment]
+1 please!

**Comment by lundyfpv** [comment]
+1

**Comment by generally-ready** [comment]
+1 (not that it seems to matter, since this PF has been open since 2024...)

**Comment by RyanEwen** [comment]
I added this a month ago - it's just sitting there waiting to be looked at, but not being looked at: https://github.com/bambulab/BambuStudio/pull/11599

**Comment by generally-ready** [comment]
> I added this a month ago - it's just sitting there waiting to be looked at, but not being looked at: [#11599](https://github.com/bambulab/BambuStudio/pull/11599)

Amazing, @RyanEwen .  By chance, does your PR also address [#11045](https://github.com/bambulab/BambuStudio/issues/11045)?  Not critical, but bloody annoying...

**Comment by RyanEwen** [comment]
> > I added this a month ago - it's just sitting there waiting to be looked at, but not being looked at: [#11599](https://github.com/bambulab/BambuStudio/pull/11599)
> 
> Amazing, [@RyanEwen](https://github.com/RyanEwen) . By chance, does your PR also address [#11045](https://github.com/bambulab/BambuStudio/issues/11045)? Not critical, but bloody annoying...

I don't think so but I haven't really compared to know. I've definitely seen that bug, but didn't think it could be arctitecture-specific. 

To be honest, I have been rebuilding as much of the Bambu Studio UI into (shameless plug) [PrintStream](https://printstream.app) and avoiding the native application as much as I possibly can.
